### ISSUE TITLE
Sequences Editor: The sequence properties is a bit "hidden" and confusing

### ISSUE THREAD

**Issue by iampremo** [YoYoGames/GameMaker-Bugs#1414]
GM-712

The properties for a sequence are "hidden" behind a button in the top right of the sequence, and opening it opens the properties box over the asset browser. This is visually messy and confusing, as the browser and the properties box use the same colours and it's difficult to tell where one ends and the other begins.

I would also say that this should NOT be hidden behind a button at the top like that, and it should be front and center in a window, like the room editor has room properties on a window/tab.

Alternatively, it might be worthwhile considering having a window pop up when you create a sequence so you can set these values before the sequence is created, and then in the future if you want to change them you can go to the properties button/box at the top right. 

Tl;dr: The sequence properties gives a poor UX.

IDE 23.1.1.65
